component/core/src/main/java/org/exoplatform/social/core/storage/api/IdentityStorage.java

```diff
@@ -412,7 +412,7 @@ default String getType(final String nodetype, final String property) {
    * @param limit
    * @return
    */
-  default public List<Identity> getIdentities(String providerId, String firstCharacterFieldName, char firstCharacter, String sortField, String sortDirection, long offset, long limit) {
+  default public List<Identity> getIdentities(String providerId, String firstCharacterFieldName, char firstCharacter, String sortField, String sortDirection, boolean excludeExternal, long offset, long limit) {
```

Comment:
Could you add a new function instead of changing the signature of an existing one

Reply:
I already changed the signature of **getIdentityDAO().getAllIdsByProviderSorted** which is inside this method . 
IMO, we also need to change the signature of this method.
WDYT ?

Reply:
I agree